zerver/lib/url_preview/preview.py

```diff
@@ -0,0 +1,63 @@
+import re
+import logging
+import traceback
+from six import text_type
+from typing import Any, Optional
+from typing.re import Match
+import requests
+from django.conf import settings
+from zerver.lib.cache import cache_with_key, get_cache_with_key
+from zerver.lib.bugdown import testing_mocks
+from .oembed import get_oembed_data
+from .parsers import OpenGraphParser, GenericParser
```

Comment:
We avoid relative imports in the Zulip project; can you use absolute imports for these?